homeassistant/components/ihc/__init__.py

```diff
@@ -128,57 +33,35 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
     return True
 
 
-def ihc_setup(hass, config, conf, controller_id):
+def ihc_setup(
+    hass: HomeAssistant,
+    config: ConfigType,
+    controller_conf: ConfigType,
+    controller_id: int,
+):
```

Comment:
```suggestion
) -> bool:
```

Style guide requires complete typing for a function that has any typing.

Reply:
Full typings added